### ISSUE TITLE
Support for Blender 4.1

### ISSUE THREAD

**Issue by firstnamewarren** [Rokoko/rokoko-studio-live-blender#117]
Do we know when the plugin will work with Blender 4.1? I have confirmed that 1.4.1 and the current 1.4.2 beta do not work for retargeting in Blender 4.1.

**Comment by Aroy-Art** [comment]
Really hope this great tool will dont die, but it feels a little bit forgotten

**Comment by czyz** [comment]
At this point (Blender 4.3.2 on macOS) the plugin won't even activate on install, it just crashes the application.

**Comment by czyz** [comment]
aha, got it installed via the in-app Extensions dialog in preferences. It fails if installed under 'Add-Ons'.

**Comment by rx-fly** [comment]
using blender 4.3, can install without issue, however it can't open a web browser to sign in, making the add-on sadly un-usefull at this time.

**Comment by rx-fly** [comment]
using blender 4.3, can install without issue, however it can't open a web browser to sign in, making the add-on sadly un-usefull at this time.

**Comment by leifniem** [comment]
Using Blender 4.5 alpha and it did open the browser just fine (Windows, Firefox)

**Comment by fast-blender-add-ons-dot-com** [comment]
anybody have issues retargeting on blender four point four

**Comment by leifniem** [comment]
> anybody have issues retargeting on blender four point four

Yup, had some in 4.5. Opened the file in 4.3 stable and the retargeting had just worked fine, the action wasn't playing in 4.5 before. Saved the file, reopened in 4.5 and it started working there too.

**Comment by fast-blender-add-ons-dot-com** [comment]
I just fixed mine for blender 4.4 if anybody's getting an issue then just
put this code here underneath that print statement at the top that that's
the code that's already in there that print statement so find it and then
put this code underneath it and it'll just work at least you know the
retargeting part I don't know about the other parts but they're retargeting
box wasn't working for me until I added this it's just a slight
workaround.......

        RetargetAnimation(bpy.types.Operator): # This is the operator that
you put the below code at the bottom of indented properly

        print('Retargeting Time:', round(time.time() - start_time, 2),
'seconds')
        # wm.progress_end()

        # Get the current action name of the target armature
        current_action = armature_target.animation_data.action

        # Store the name of the current action
        current_action_name = current_action.name if current_action else
None

        # Find a different action to switch to temporarily
        # You can pick any other action that exists in the armature's
action list
        other_action = None
        for action in bpy.data.actions:
            if action.name != current_action_name:
                other_action = action
                break  # Get the first other action available

        # If a different action was found, switch to it temporarily
        if other_action:
            armature_target.animation_data.action = other_action
            # Now switch back to the original action to trigger the refresh
            armature_target.animation_data.action = bpy.data.actions.get
(current_action_name)

            # Print out which actions are being switched to debug
            print(f"Switched to action: {other_action.name} and then back
to: {current_action_name}")
            print(f"...(To invoke Blenders upgrading of the characters
action system to the new slot system.)")
        else:
            # If no other action is found, print a message indicating that
            print("No other action found to switch to.")

**Comment by Darkblader24** [comment]
We have now added general Blender 4.4 support!

**Comment by fast-blender-add-ons-dot-com** [comment]
Thank you very much I would just say between you and me as a business owner
I would always make sure my software is updated immediately as the new
version comes out just an idea that that way you know you're going to look
a lot cooler to all the people that use your software... Because you know
tons of people rely on this so when it's not available for the latest
version of blender when a new feature comes out that they want to try you
know that's a major disappointment
